LegitimateURLShortener.txt

```diff
@@ -884,7 +883,7 @@ $removeparam=_kx
 
 ! https://www.affluent.io/affiliate-marketing-blog/affiliate-sub-campaign-sid-tracking-guide/
 ! https://strackr.com/subid
-$removeparam=AFFSRC
+$removeparam=/^affsrc=/i
```

Comment:
@git-101-collab Why should this be modified? As I see it, these two rules are equivalent.

Reply:
We had rules for `affsrc`, `AFFSRC` and  I found `Affsrc`
(`https://www.tempurpedic.com/shop-mattresses/black-friday-sitebuster/v/3189/?Affsrc=1`)

I tried to just use `$removeparam=/^affsrc=/i` to cover all of them in one rule